tempus_fugit_minecraft/window.py

```diff
@@ -359,7 +359,6 @@ def draw_pause_menu(self) -> None:
             quit buttons.
         @see [Issue#22](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/22)
         """
-    def draw_pause_menu(self) -> None:
```

Comment:
@criticalglitch As you have mentioned earlier. this what was causing the problem.